### DISCUSSION TITLE
CoolWallet Coin Tester ERROR

### DISCUSSION THREAD

**Discussion by fantersee** [CoolBitX-Technology/coolwallet-js-sdk#267] [Q&A]
-  Description
   執行 coin-template 的 Get address 出現錯誤訊息，這是代表什麼地方出錯?
 
-  Screenshots or appendix
![S__275881986](https://user-images.githubusercontent.com/4178903/151295576-8ef5c142-be01-4316-b0b3-e73be739a990.jpg)

**Comment by hankliu418** [comment]
請問是否有先做過set seed了呢？

**Comment by fantersee** [comment]
set seed 後問題排除 感謝

Hank ***@***.***> 於 2022年1月28日 週五 上午11:24寫道：
